python/ray/autoscaler/azure/example-full.yaml

```diff
@@ -164,9 +164,8 @@ setup_commands: []
     # - pip install -U "ray[default] @ https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp38-cp38-manylinux2014_x86_64.whl"
 
 # Custom commands that will be run on the head node after common setup.
-# NOTE: rayproject/ray-ml:latest has azure packages bundled
 head_setup_commands: []
-    # - pip install -U azure-cli-core==2.22.0 azure-mgmt-compute==14.0.0 azure-mgmt-msi==1.0.0 azure-mgmt-network==10.2.0 azure-mgmt-resource==13.0.0
+    - pip install -U azure-cli-core==2.22.0 azure-mgmt-compute==14.0.0 azure-mgmt-msi==1.0.0 azure-mgmt-network==10.2.0 azure-mgmt-resource==13.0.0
```

Comment:
is this 2.22.0 what we expect? in docker/base-deps/Dockerfile we deleted a 2.40.0

Reply:
I think even azure-cli-core==2.62.0 is too old since it brings an old paramiko==2.12.0. What about we use the latest version now?

Reply:
I would guess this thing is 70% chance broken already even without the change.

I just want to remove the ones in the base image, and I am uncommenting this line just as a courtesy. please feel free to changes these to the right version core team thinks this should be, but maybe in a follow up?

or let me the exact line I should put here, and I will just copy it here..

Reply:
another idea can be, I can just drop all the versions constraints.. and let pip figure it out by itself...

Reply:
ok, I am changing it to be consistent with https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/azure/defaults.yaml#L139

if that one does not work, I am claiming that it is not my fault..

Reply:
cool. We can also ask @gramhagen to take a look